sdk/servicebus/service-bus/changelog.md

```diff
@@ -1,3 +1,8 @@
+# 2019-09-19 1.1.0
+
+- Update library to support running all Service Bus operations in browser.
+[PR 5128](https://github.com/Azure/azure-sdk-for-js/pull/5128) related to [issue 3373](https://github.com/Azure/azure-sdk-for-js/issues/3373)
```

Comment:
Add a note about not supporting AAD in browser yet and that using a connection string is the only way to use the library